packages/fury-adapter-oas3-parser/test/unit/parser/oas/parseResponseObject-test.js

```diff
@@ -36,19 +36,32 @@ describe('Response Object', () => {
       expect(result.length).to.equal(1);
       expect(result).to.contain.warning("'Response Object' response status code ranges are unsupported");
     });
+  });
+
+  describe('description', () => {
+    it('exposes description of response', () => {
```

Comment:
This test isn't testing that the description in found within the response and the implementation isn't exposing the copy element.

We will need to push the copy element into the HttpResponse's in `parseResponseObject.js`, on line `~80` and `~87` where the status code is added to the HttpResponse object we should insert the copy element into these elements (at index 0). There is similiar code for this in `lib/parser/oas/parseOperationObject.js`:

```js
      const description = operation.get('description');
      if (description) {
        transition.push(description);
      }
```

In our case this will end up in multiple responses and thus we should make a `clone` of the copy element (`httpResponse.push(description.clone);`). That because it isn't good practise to put the exact same element instance multiple times in the tree as any consumers of the parser may do mutation which may incorrectly happen to other references to the same copy element.

The biggest reason though is that a consumer might use `.freeze()` on the element tree to make it immutable (and also add .parent propert accessors which falls down when the same element is reused because it can only have one parent element). (https://github.com/refractproject/minim/blob/1b9d9ee66689839cd9a5246741e0d10a7a945402/lib/primitives/element.js#L34)

Then we'll see the test fixtures include the copy elements (can be generated with `npm run test:integration:fix`).